src/mudlet.cpp

```diff
@@ -499,8 +499,6 @@ mudlet::mudlet()
     connect(dactionShowMap, &QAction::triggered, this, &mudlet::slot_mapper);
     connect(dactionOptions, &QAction::triggered, this, &mudlet::slot_show_options_dialog);
     connect(dactionAbout, &QAction::triggered, this, &mudlet::slot_show_about_dialog);
-    // PLACEMARKER: Save for later restoration (2 of 2) (by adding a "Close" (profile) option to first menu on menu bar:
```

Comment:
👎 I do want to put that close option (back?) in - IMHO it logically make sense, though it will need some checking that the Application does return to the state prior to the initial "Connect Dialog" appearance on start-up. Remember when running a single profile there is not method currently to close the profile *without* terminating the application (perhaps we might want to use `<CTRL>+<F4>`) even though that is not an unreasonable way of operating.

OTOH for consistency I do not see you deleting the other `// PLACEMARKER: Save for later restoration...` for this, even though I made it clear that there is two of them! 😮

Reply:
It can be re-added when needed later on, it's always available in Git history!